doc/tools/coqrst/coqdomain.py

```diff
@@ -895,14 +900,19 @@ def add_coq_output_1(self, repl, node):
         if options['reset']:
             repl.sendone("Reset Initial.")
             repl.sendone("Set Coqtop Exit On Error.")
+            repl.sendone("Set Warnings \"+default\".")
```

Comment:
You should also send it at the start (L926).
Perhaps we should lift this (re)initialisation stuff to a function.